Cargo.toml

```diff
@@ -12,7 +12,7 @@ bytecount = "0.6"
 clap = { version = "4.3.0", features = ["derive"] }
 encoding_rs = "0.8.14"
 encoding_rs_io = "0.1.6"
-ignore = { git = "https://github.com/helixbass/ripgrep", rev = "e811564" }
+ignore = { git = "https://github.com/helixbass/ripgrep", rev = "012f2f1" }
```

Comment:
The existing known file-types in `ignore` only included `*.scm` under `"lisp"` (whose globs overlapped with Elisp) so I added an explicit file-type for tree-sitter-queries to avoid that being an issue